### PR TITLE
Remove default code generators & clarify connection to Scratch VM

### PR DESCRIPTION
**Pull request by thisandagain**
## Changes
- Removes the "default" code generators from Scratch Blocks (dart, php, python, lua, javascript)
- Update the `README.md` to clarify connection between Scratch Blocks & [Scratch VM](https://github.com/llk/scratch-vm)

## Notes
- Maintains support for generators in general (does not remove `core/generator.js` or related tests) but attempts to take a step towards clarifying the recommended pathway for developers working with Scratch Blocks

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -36,3 +43,4 @@ common.pyc
 /gh-pages/_site
 /*compiler*.jar
 /local_blockly_compressed_vertical.js
+/chromedriver
```
Comment (thisandagain) [inline .gitignore:46]: This is one minor addition to clean-up an ergonomics issue. Happy to split this out if you prefer @picklesrus @rschamp.

**Review by rschamp — APPROVED** [review]
This LGTM. But I defer to @picklesrus in case there's something about building scratch-blocks I'm not considering.

**Comment by towerofnix** [comment]
This PR is definitely appreciated.. I've had the `gen_generator` lines commented out on my own branches for a very (very) long time! I've always had to make sure I don't accidentally `git add build.py`. :P

**Comment by picklesrus** [comment]
LGTM - this is essentially the opposite of what I did in #787. 
There is still more generator code throughout the repo, but seems fine for that to get cleaned up later.

**Comment by PeaceInDream** [comment]
if i want to restart the generator for the Arduino , what should i do ?

**Pull request merged**